test/contlinear.jl

```diff
@@ -356,6 +356,68 @@ function contlineartest(solver::MOI.AbstractSolver, ε=Base.rtoldefault(Float64)
         end
     end
 
+    @testset "Issue #40 from Gurobi.jl" begin
+        # min  x
+        # s.t. x >= 0
+        #      x >= 3
+
+        m = MOI.SolverInstance(solver)
+
+        x = MOI.addvariables!(m)
+        @test MOI.getattribute(m, MOI.NumberOfVariables()) == 1
+
+        MOI.addconstraint!(m, MOI.ScalarVariablewiseFunction(x), MOI.GreaterThan(0.0))
+        cf = MOI.ScalarAffineFunction([x], [1.0], 0.0)
+        MOI.addconstraint!(m, cf, MOI.GreaterThan(3.0))
+
+        @test MOI.getattribute(m, MOI.NumberOfConstraints{MOI.ScalarVariablewiseFunction,MOI.GreaterThan{Float64}}()) == 1
+        @test MOI.getattribute(m, MOI.NumberOfConstraints{MOI.ScalarAffineFunction{Float64},MOI.GreaterThan{Float64}}()) == 1
+
+        objf = MOI.ScalarVariablewiseFunction(x)
+        MOI.setobjective!(m, MOI.MinSense, objf)
```

Comment:
I also feel that it's unnecessary to test ``ScalarVariablewiseFunction`` as an objective function. `ScalarAffineFunction` is sufficient in terms of expressiveness in the context of LPs.